connector/src/main/scala/com/microsoft/kusto/spark/common/KustoOptions.scala

```diff
@@ -25,31 +30,34 @@ trait KustoOptions {
 
   val KUSTO_ACCESS_TOKEN: String = newOption("accessToken")
 
-  // Use only for local runs where one can open the browser
+  // Use only for local runs where one can open the browser - set to true to enable
   val KUSTO_USER_PROMPT: String = newOption("userPrompt")
   // AAD application pfx certificate path
   val KUSTO_AAD_APP_CERTIFICATE_PATH: String = newOption("kutoAadAppCertPath")
   // AAD application certificate password
   val KUSTO_AAD_APP_CERTIFICATE_PASSWORD: String = newOption("kutoAadAppCertPassword")
 
+  // KeyVault options. Relevant only if credentials need to be retrieved from Key Vault
+  val KEY_VAULT_URI = "keyVaultUri"
```

Comment:
Too late to change this now, but for the record this is a URL. URI is overly broad.

Reply:
again - true - but this is not how it is called everywhere ..